aws_quickstart/main_v2.yaml

```diff
@@ -95,6 +136,18 @@ Conditions:
       - GovCloud
 Resources:
   # A Macro used to generate policies for the integration IAM role based on user inputs
+  DatadogAgentlessScanning:
+    Type: AWS::CloudFormation::Stack
+    Condition: EnableAgentlessScanning
+    Properties:
+      TemplateURL: "https://<BUCKET_PLACEHOLDER>.s3.amazonaws.com/aws/datadog_agentless_scanning.yaml"
+      Parameters:
+        DatadogAPIKey: !Ref APIKey
+        DatadogAPPKey: !Ref APPKey
+        DatadogSite: !Ref DatadogSite
```

Comment:
Does this work in Fed? This template now can be called in fed.

Reply:
It does not. I just added a check to not deploy the Agentless scanning CloudFormation stack if the DatadogSite is set to ddog-gov.com.

Reply:
I think ideally we'd do this using a CloudFormation Rule so that the template fails immediately and the customer can change their configuration: https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/rules-section-structure.html
Also on the fed web-ui side are we just not showing these options?
This is just a nit, so we can address this later as well.